scriptmodules/ports/steamlink.sh

```diff
@@ -33,4 +33,12 @@ function remove_steamlink() {
 
 function configure_steamlink() {
     addPort "$md_id" "steamlink" "Steam Link" "/usr/bin/steamlink"
+
+    # RetroPie sets overscan by default, which requires an override
+    if egrep '^overscan_scale=1' /boot/config.txt >/dev/null 2>&1; then
```

Comment:
I don't think grep -E is needed.

also `&>/dev/null` would be more concise in bash.

Reply:
even better: `grep -q '^overscan_scale=1' /boot/config.txt` doesn't produce output at all ;)

Reply:
it will if config.txt doesn't exist.

Reply:
right, sorry, I keep forgetting that RetroPie doesn't run only on RPIs :)

Reply:
This module is RPI only, but actually on a RPI config.txt may not be present. Boot partition may not be mounted or like with berryboot.